01-shapefile-attributes.Rmd

```diff
@@ -1,15 +1,18 @@
 ---
 layout: post
-title: "Lesson 01: Work With Shapefile Attributes in R"
+title: "Lesson 01: Shapefile Metadata & Attributes in R"
```

Comment:
Title:  Does the metadata and attribute lessons here apply to both raster and vector data?  If not, add vector to title?  If so, links from Spatial Raster?